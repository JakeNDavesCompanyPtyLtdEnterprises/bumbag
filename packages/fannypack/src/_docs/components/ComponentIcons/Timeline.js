import React from 'react';

const SvgComponent = props => (
  <svg width={120} height={60} viewBox="0 0 120 60" fill="none">
    <path
      d="M0 4C0 1.79086 1.79086 0 4 0H116C118.209 0 120 1.79086 120 4V56C120 58.2091 118.209 60 116 60H4C1.79086 60 0 58.2091 0 56V4Z"
      fill="#fff"
    />
    <rect x="30" y="6" width="28" height="3.65" fill="#E7E5F4" />
    <rect x="30" y="14" width="60" height="3.65" fill="#E7E5F4" />
    <rect x="30" y="25" width="28" height="3.65" fill="#E7E5F4" />
    <rect x="30" y="33" width="60" height="3.65" fill="#E7E5F4" />
    <rect x="30" y="44" width="28" height="3.65" fill="#E7E5F4" />
    <rect x="30" y="52" width="60" height="3.65" fill="#E7E5F4" />
    <line x1="20.5" y1="12" x2="20.5" y2="30" stroke="#3926A5" />
    <line x1="20.5" y1="32" x2="20.5" y2="50" stroke="#E7E5F4" />
    <circle cx="20.5" cy="11.5" r="2.5" fill="#3926A5" />
    <circle cx="20.5" cy="30.5" r="2.5" fill="#3926A5" />
    <circle cx="20.5" cy="49.5" r="2.5" fill="#E7E5F4" />
  </svg>
);

export default SvgComponent;
