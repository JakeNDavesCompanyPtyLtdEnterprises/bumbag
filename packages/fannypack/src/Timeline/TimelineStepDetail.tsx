/* eslint react/prop-types: 0 */
import * as React from 'react';
import * as PropTypes from 'prop-types';
import { ParagraphProps as ReakitParagraphProps } from 'reakit/ts/Paragraph/Paragraph';

import { TimelineStepDetail as _TimelineStepDetail } from './styled';

export type LocalTimelineStepDetailProps = {
  children: React.ReactNode;
  color?: string;
};
export type TimelineStepDetailProps = ReakitParagraphProps & LocalTimelineStepDetailProps;

const TimelineStepDetail: React.FunctionComponent<LocalTimelineStepDetailProps> = ({ children, color, ...props }) => (
  <_TimelineStepDetail color={color} {...props}>
    {children}
  </_TimelineStepDetail>
);

TimelineStepDetail.propTypes = {
  children: PropTypes.node.isRequired,
  color: PropTypes.string
};

TimelineStepDetail.defaultProps = {
  color: undefined
};

const C: React.FunctionComponent<TimelineStepDetailProps> = TimelineStepDetail;
export default C;
