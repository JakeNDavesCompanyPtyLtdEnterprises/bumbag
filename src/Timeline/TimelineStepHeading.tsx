/* eslint react/prop-types: 0 */
import * as React from 'react';
import * as PropTypes from 'prop-types';
import { ParagraphProps as ReakitParagraphProps } from 'reakit/ts/Paragraph/Paragraph';

import { TimelineStepHeading as _TimelineStepHeading } from './styled';

export type LocalTimelineStepHeadingProps = {
  children: React.ReactNode;
  color?: string;
};
export type TimelineStepHeadingProps = ReakitParagraphProps & LocalTimelineStepHeadingProps;

const TimelineStepHeading: React.FunctionComponent<LocalTimelineStepHeadingProps> = ({ children, color, ...props }) => (
  <_TimelineStepHeading color={color} fontWeight="semibold" {...props}>
    {children}
  </_TimelineStepHeading>
);

TimelineStepHeading.propTypes = {
  children: PropTypes.node.isRequired,
  color: PropTypes.string
};

TimelineStepHeading.defaultProps = {
  color: undefined
};

const C: React.FunctionComponent<TimelineStepHeadingProps> = TimelineStepHeading;
export default C;
