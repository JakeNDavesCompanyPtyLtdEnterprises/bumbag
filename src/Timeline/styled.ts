import { theme, palette } from 'styled-tools';

import styled, { space } from '../styled';
import { Box } from '../primitives';
import { TimelineStepProps, TimelineRowProps } from './TimelineStep';
import { TimelineStepDetailProps } from './TimelineStepDetail';
import { TimelineStepHeadingProps } from './TimelineStepHeading';
// @ts-ignore
import Paragraph from '../Paragraph';

export const Timeline = styled(Box)`
  & {
    ${theme('fannypack.Timeline.base')};
  }
`;

export const TimelineRow = styled(Box)<TimelineRowProps>`
  display: flex;
  position: relative;

  &:last-child {
    margin-bottom: 0;
  }

  &::before {
    content: '';
    border: 1px solid ${props => palette(props.lineColor || 'textTint', props.lineColor)};
    position: absolute;
    height: 100%;
    left: ${space(1)}rem;
    margin-top: ${space(5)}rem;
  }

  &:last-child::before {
    border: 0;
  }
`;

export const TimelineStep = styled(Box)<TimelineStepProps>`
  & {
    ${theme('fannypack.Timeline.Step.base')};
  }
`;

export const TimelineBullet = styled(Box)`
  border-radius: 50%;
  display: inline-flex;
  flex-shrink: 0;
  height: 0.625rem;
  width: 0.625rem;
  margin-top: ${space(4)}rem;
  margin-right: ${space(2)}rem;
  z-index: 1;

  & {
    ${theme('fannypack.Timeline.Bullet.base')};
  }
`;

export const TimelineStepHeading = styled(Paragraph)<TimelineStepHeadingProps>`
  margin-top: ${space(2)}rem;
  margin-bottom: ${space(1)}rem;
  line-height: 1.5rem;
  color: ${props => props.color || palette('text')};
  font-weight: ${theme('fannypack.fontWeights.semibold')};

  &:not(:last-child) {
    margin-top: ${space(2)}rem;
    margin-bottom: ${space(1)}rem;
  }

  & {
    ${theme('fannypack.Timeline.StepHeading.base')};
  }
`;

export const TimelineStepDetail = styled(Box)<TimelineStepDetailProps>`
  color: ${props => props.color || palette('text')};

  & {
    ${theme('fannypack.Timeline.StepDetail.base')};
  }
`;
