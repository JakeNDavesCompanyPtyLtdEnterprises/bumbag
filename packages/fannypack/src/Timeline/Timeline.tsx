/* eslint react/prop-types: 0 */
import * as React from 'react';
import * as PropTypes from 'prop-types';
// @ts-ignore
import _isArrayLikeObject from 'lodash/isArrayLikeObject';

import TimelineStep, { TimelineStepProps } from './TimelineStep';
import TimelineStepDetail, { TimelineStepDetailProps } from './TimelineStepDetail';
import TimelineStepHeading, { TimelineStepHeadingProps } from './TimelineStepHeading';
import { Timeline as _Timeline } from './styled';

export type LocalTimelineProps = {
  activeStep?: string;
  children: React.ReactNode;
  progressColor?: string;
};
export type TimelineProps = LocalTimelineProps;
export type TimelineComponents = {
  Step: React.FunctionComponent<TimelineStepProps>;
  StepHeading: React.FunctionComponent<TimelineStepHeadingProps>;
  StepDetail: React.FunctionComponent<TimelineStepDetailProps>;
};

export const Timeline: React.FunctionComponent<LocalTimelineProps> & TimelineComponents = ({
  activeStep,
  children,
  progressColor
}) => {
  if (!_isArrayLikeObject(children)) {
    return null;
  }
  const activeIndexStep = React.Children.toArray(children).findIndex(
    (child: any) => child.props.a11yTitleId === activeStep
  );
  return (
    <_Timeline>
      {React.Children.map(children, (child, index) => {
        const _child = child as React.ReactElement<any>;
        const color = progressColor ? progressColor : _child.props.progressColor;
        return index <= activeIndexStep
          ? React.cloneElement(_child, {
              lineColor: index < activeIndexStep ? color : undefined,
              progressColor: color || undefined
            })
          : child;
      })}
    </_Timeline>
  );
};

Timeline.Step = TimelineStep;
Timeline.StepHeading = TimelineStepHeading;
Timeline.StepDetail = TimelineStepDetail;

Timeline.propTypes = {
  activeStep: PropTypes.string,
  children: PropTypes.node.isRequired,
  progressColor: PropTypes.string
};
Timeline.defaultProps = {
  activeStep: undefined,
  progressColor: undefined
};

// @ts-ignore
const C: React.FunctionComponent<TimelineProps> & TimelineComponents = Timeline;
export default C;
