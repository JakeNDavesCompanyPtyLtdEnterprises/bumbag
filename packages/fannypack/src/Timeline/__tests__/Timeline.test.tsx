import * as React from 'react';
import render from '../../_utils/tests/render';
import Timeline from '../Timeline';

describe('Timeline', () => {
  it('renders correctly', () => {
    const { container } = render(
      <Timeline>
        <Timeline.Step a11yTitleId="StepHeading1" a11yDescriptionId="StepDetail1">
          <Timeline.StepHeading id="StepHeading1">Step one Heading</Timeline.StepHeading>
          <Timeline.StepDetail id="StepDetail1">Step one details goes here</Timeline.StepDetail>
        </Timeline.Step>
        <Timeline.Step a11yTitleId="StepHeading2" a11yDescriptionId="StepDetail2">
          <Timeline.StepHeading id="StepHeading2">Step two Heading</Timeline.StepHeading>
          <Timeline.StepDetail id="StepDetail2">Step two details goes here</Timeline.StepDetail>
        </Timeline.Step>
        <Timeline.Step a11yTitleId="StepHeading3" a11yDescriptionId="StepDetail3">
          <Timeline.StepHeading id="StepHeading3">Step three Heading</Timeline.StepHeading>
          <Timeline.StepDetail id="StepDetail3">Step three details goes here</Timeline.StepDetail>
        </Timeline.Step>
      </Timeline>
    );
    expect(container.firstChild).toMatchSnapshot();
  });

  describe('with custom color in step', () => {
    it('renders correctly', () => {
      const { container } = render(
        <Timeline>
          <Timeline.Step a11yTitleId="StepHeading1" a11yDescriptionId="StepDetail1">
            <Timeline.StepHeading id="StepHeading1">Step one Heading</Timeline.StepHeading>
            <Timeline.StepDetail id="StepDetail1">Step one details goes here</Timeline.StepDetail>
          </Timeline.Step>
          <Timeline.Step
            progressColor="green"
            lineColor="green"
            a11yTitleId="StepHeading2"
            a11yDescriptionId="StepDetail2"
          >
            <Timeline.StepHeading id="StepHeading2">Step two Heading</Timeline.StepHeading>
            <Timeline.StepDetail id="StepDetail2">Step two details goes here</Timeline.StepDetail>
          </Timeline.Step>
          <Timeline.Step a11yTitleId="StepHeading3" a11yDescriptionId="StepDetail3">
            <Timeline.StepHeading id="StepHeading3">Step three Heading</Timeline.StepHeading>
            <Timeline.StepDetail id="StepDetail3">Step three details goes here</Timeline.StepDetail>
          </Timeline.Step>
        </Timeline>
      );
      expect(container.firstChild).toMatchSnapshot();
    });
  });

  describe('with activeStep', () => {
    it('renders correctly', () => {
      const { container } = render(
        <Timeline activeStep="StepHeading2" progressColor="purple">
          <Timeline.Step a11yTitleId="StepHeading1" a11yDescriptionId="StepDetail1">
            <Timeline.StepHeading id="StepHeading1">Step one Heading</Timeline.StepHeading>
            <Timeline.StepDetail id="StepDetail1">Step one details goes here</Timeline.StepDetail>
          </Timeline.Step>
          <Timeline.Step a11yTitleId="StepHeading2" a11yDescriptionId="StepDetail2">
            <Timeline.StepHeading id="StepHeading2">Step two Heading</Timeline.StepHeading>
            <Timeline.StepDetail id="StepDetail2">Step two details goes here</Timeline.StepDetail>
          </Timeline.Step>
          <Timeline.Step a11yTitleId="StepHeading3" a11yDescriptionId="StepDetail3">
            <Timeline.StepHeading id="StepHeading3">Step three Heading</Timeline.StepHeading>
            <Timeline.StepDetail id="StepDetail3">Step three details goes here</Timeline.StepDetail>
          </Timeline.Step>
        </Timeline>
      );
      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
