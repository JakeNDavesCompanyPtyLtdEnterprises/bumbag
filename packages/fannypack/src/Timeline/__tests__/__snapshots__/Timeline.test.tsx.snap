// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timeline renders correctly 1`] = `
.c0 {
  margin: unset;
  padding: unset;
  border: unset;
  background: unset;
  font: unset;
  font-family: inherit;
  font-size: 100%;
  box-sizing: border-box;
  background-color: unset;
  color: inherit;
}

.c0:focus:not(:focus-visible) {
  outline: none;
}

.c3 {
  margin: unset;
  padding: unset;
  border: unset;
  background: unset;
  font: unset;
  font-family: inherit;
  font-size: 100%;
  box-sizing: border-box;
  background-color: unset;
  color: inherit;
  background-color: #d9dee2 !important;
}

.c3:focus:not(:focus-visible) {
  outline: none;
}

.c5:not(:last-child) {
  margin-bottom: 1rem;
}

.c5 .c7 {
  top: 0.15em;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  position: relative;
}

.c1:last-child {
  margin-bottom: 0;
}

.c1::before {
  content: '';
  border: 1px solid #d9dee2;
  position: absolute;
  height: 100%;
  left: 0.25rem;
  margin-top: 1.25rem;
}

.c1:last-child::before {
  border: 0;
}

.c2 {
  border-radius: 50%;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  height: 0.625rem;
  width: 0.625rem;
  margin-top: 1rem;
  margin-right: 0.5rem;
  z-index: 1;
}

.c4 {
  margin-top: 0.5rem;
  margin-bottom: 0.25rem;
  line-height: 1.5rem;
  color: #435a6f;
  font-weight: 600;
}

.c4:not(:last-child) {
  margin-top: 0.5rem;
  margin-bottom: 0.25rem;
}

.c6 {
  color: #435a6f;
}

<div
  class="c0"
>
  <div
    aria-describedby="StepDetail1"
    aria-labelledby="StepHeading1"
    class="c1 c0"
  >
    <div
      class="c2 c3"
    />
    <div
      aria-describedby="StepDetail1"
      aria-labelledby="StepHeading1"
      class="c0"
    >
      <p
        class="c4 c5 c0"
        id="StepHeading1"
      >
        Step one Heading
      </p>
      <div
        class="c6 c0"
        id="StepDetail1"
      >
        Step one details goes here
      </div>
    </div>
  </div>
  <div
    aria-describedby="StepDetail2"
    aria-labelledby="StepHeading2"
    class="c1 c0"
  >
    <div
      class="c2 c3"
    />
    <div
      aria-describedby="StepDetail2"
      aria-labelledby="StepHeading2"
      class="c0"
    >
      <p
        class="c4 c5 c0"
        id="StepHeading2"
      >
        Step two Heading
      </p>
      <div
        class="c6 c0"
        id="StepDetail2"
      >
        Step two details goes here
      </div>
    </div>
  </div>
  <div
    aria-describedby="StepDetail3"
    aria-labelledby="StepHeading3"
    class="c1 c0"
  >
    <div
      class="c2 c3"
    />
    <div
      aria-describedby="StepDetail3"
      aria-labelledby="StepHeading3"
      class="c0"
    >
      <p
        class="c4 c5 c0"
        id="StepHeading3"
      >
        Step three Heading
      </p>
      <div
        class="c6 c0"
        id="StepDetail3"
      >
        Step three details goes here
      </div>
    </div>
  </div>
</div>
`;

exports[`Timeline with activeStep renders correctly 1`] = `
.c0 {
  margin: unset;
  padding: unset;
  border: unset;
  background: unset;
  font: unset;
  font-family: inherit;
  font-size: 100%;
  box-sizing: border-box;
  background-color: unset;
  color: inherit;
}

.c0:focus:not(:focus-visible) {
  outline: none;
}

.c7 {
  margin: unset;
  padding: unset;
  border: unset;
  background: unset;
  font: unset;
  font-family: inherit;
  font-size: 100%;
  box-sizing: border-box;
  background-color: unset;
  color: inherit;
  background-color: #d9dee2 !important;
}

.c7:focus:not(:focus-visible) {
  outline: none;
}

.c4:not(:last-child) {
  margin-bottom: 1rem;
}

.c4 .c8 {
  top: 0.15em;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  position: relative;
}

.c6:last-child {
  margin-bottom: 0;
}

.c6::before {
  content: '';
  border: 1px solid #d9dee2;
  position: absolute;
  height: 100%;
  left: 0.25rem;
  margin-top: 1.25rem;
}

.c6:last-child::before {
  border: 0;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  position: relative;
}

.c1:last-child {
  margin-bottom: 0;
}

.c1::before {
  content: '';
  border: 1px solid purple;
  position: absolute;
  height: 100%;
  left: 0.25rem;
  margin-top: 1.25rem;
}

.c1:last-child::before {
  border: 0;
}

.c2 {
  border-radius: 50%;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  height: 0.625rem;
  width: 0.625rem;
  margin-top: 1rem;
  margin-right: 0.5rem;
  z-index: 1;
}

.c3 {
  margin-top: 0.5rem;
  margin-bottom: 0.25rem;
  line-height: 1.5rem;
  color: #435a6f;
  font-weight: 600;
}

.c3:not(:last-child) {
  margin-top: 0.5rem;
  margin-bottom: 0.25rem;
}

.c5 {
  color: #435a6f;
}

<div
  class="c0"
>
  <div
    aria-describedby="StepDetail1"
    aria-labelledby="StepHeading1"
    class="c1 c0"
  >
    <div
      class="c2 c0"
      style="background-color: purple;"
    />
    <div
      aria-describedby="StepDetail1"
      aria-labelledby="StepHeading1"
      class="c0"
    >
      <p
        class="c3 c4 c0"
        id="StepHeading1"
      >
        Step one Heading
      </p>
      <div
        class="c5 c0"
        id="StepDetail1"
      >
        Step one details goes here
      </div>
    </div>
  </div>
  <div
    aria-describedby="StepDetail2"
    aria-labelledby="StepHeading2"
    class="c6 c0"
  >
    <div
      class="c2 c0"
      style="background-color: purple;"
    />
    <div
      aria-describedby="StepDetail2"
      aria-labelledby="StepHeading2"
      class="c0"
    >
      <p
        class="c3 c4 c0"
        id="StepHeading2"
      >
        Step two Heading
      </p>
      <div
        class="c5 c0"
        id="StepDetail2"
      >
        Step two details goes here
      </div>
    </div>
  </div>
  <div
    aria-describedby="StepDetail3"
    aria-labelledby="StepHeading3"
    class="c6 c0"
  >
    <div
      class="c2 c7"
    />
    <div
      aria-describedby="StepDetail3"
      aria-labelledby="StepHeading3"
      class="c0"
    >
      <p
        class="c3 c4 c0"
        id="StepHeading3"
      >
        Step three Heading
      </p>
      <div
        class="c5 c0"
        id="StepDetail3"
      >
        Step three details goes here
      </div>
    </div>
  </div>
</div>
`;

exports[`Timeline with custom color in step renders correctly 1`] = `
.c0 {
  margin: unset;
  padding: unset;
  border: unset;
  background: unset;
  font: unset;
  font-family: inherit;
  font-size: 100%;
  box-sizing: border-box;
  background-color: unset;
  color: inherit;
}

.c0:focus:not(:focus-visible) {
  outline: none;
}

.c3 {
  margin: unset;
  padding: unset;
  border: unset;
  background: unset;
  font: unset;
  font-family: inherit;
  font-size: 100%;
  box-sizing: border-box;
  background-color: unset;
  color: inherit;
  background-color: #d9dee2 !important;
}

.c3:focus:not(:focus-visible) {
  outline: none;
}

.c5:not(:last-child) {
  margin-bottom: 1rem;
}

.c5 .c8 {
  top: 0.15em;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  position: relative;
}

.c1:last-child {
  margin-bottom: 0;
}

.c1::before {
  content: '';
  border: 1px solid #d9dee2;
  position: absolute;
  height: 100%;
  left: 0.25rem;
  margin-top: 1.25rem;
}

.c1:last-child::before {
  border: 0;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  position: relative;
}

.c7:last-child {
  margin-bottom: 0;
}

.c7::before {
  content: '';
  border: 1px solid green;
  position: absolute;
  height: 100%;
  left: 0.25rem;
  margin-top: 1.25rem;
}

.c7:last-child::before {
  border: 0;
}

.c2 {
  border-radius: 50%;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  height: 0.625rem;
  width: 0.625rem;
  margin-top: 1rem;
  margin-right: 0.5rem;
  z-index: 1;
}

.c4 {
  margin-top: 0.5rem;
  margin-bottom: 0.25rem;
  line-height: 1.5rem;
  color: #435a6f;
  font-weight: 600;
}

.c4:not(:last-child) {
  margin-top: 0.5rem;
  margin-bottom: 0.25rem;
}

.c6 {
  color: #435a6f;
}

<div
  class="c0"
>
  <div
    aria-describedby="StepDetail1"
    aria-labelledby="StepHeading1"
    class="c1 c0"
  >
    <div
      class="c2 c3"
    />
    <div
      aria-describedby="StepDetail1"
      aria-labelledby="StepHeading1"
      class="c0"
    >
      <p
        class="c4 c5 c0"
        id="StepHeading1"
      >
        Step one Heading
      </p>
      <div
        class="c6 c0"
        id="StepDetail1"
      >
        Step one details goes here
      </div>
    </div>
  </div>
  <div
    aria-describedby="StepDetail2"
    aria-labelledby="StepHeading2"
    class="c7 c0"
  >
    <div
      class="c2 c0"
      style="background-color: green;"
    />
    <div
      aria-describedby="StepDetail2"
      aria-labelledby="StepHeading2"
      class="c0"
    >
      <p
        class="c4 c5 c0"
        id="StepHeading2"
      >
        Step two Heading
      </p>
      <div
        class="c6 c0"
        id="StepDetail2"
      >
        Step two details goes here
      </div>
    </div>
  </div>
  <div
    aria-describedby="StepDetail3"
    aria-labelledby="StepHeading3"
    class="c1 c0"
  >
    <div
      class="c2 c3"
    />
    <div
      aria-describedby="StepDetail3"
      aria-labelledby="StepHeading3"
      class="c0"
    >
      <p
        class="c4 c5 c0"
        id="StepHeading3"
      >
        Step three Heading
      </p>
      <div
        class="c6 c0"
        id="StepDetail3"
      >
        Step three details goes here
      </div>
    </div>
  </div>
</div>
`;
